app/server.js

```diff
@@ -9,51 +15,51 @@ import {
   joiErrors,
 } from './middlewares/errors';
 import jsonReplacer from './utils/jsonReplacer';
-import { ensureAuthentificated } from './middlewares/authentification';
-import dotenv from 'dotenv';
+import { ensureAuthenticated } from './middlewares/authentification';
```

Comment:
change the file name to `authentication`

Reply:
Done